_pages/en/strategy-alternate-model.md

```diff
@@ -0,0 +1,360 @@
+    Technical debt is a stakeholder because they are working on an evergreen funding model (e.g. tax)
+    </td>
+    <td>
+    CFOB<br>
+    EPMO<br>
+    Technical Debt
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="4">Architecture</td>
+    <td>Get endorsement from EA</td>
+    <td>Engage with Enterprise Architecture, specifically Business Architecture, Solutions Architecture, and Technical Architecture teams to seek endorsement on the Alternate Delivery Model and their contribution</td>
+    <td>
+    EA<br>
+    ARC<br>
+    Interoperability<br>
+    IITB Senior Advisors<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Review technical brick process</td>
+    <td>Review technical bricks management process to allow IT Products team to add new standards (select their own technical stacks)</td>
+    <td>Technical Architecture</td>
+  </tr>
+  <tr>
+    <td>Define API approval process</td>
+    <td>Formally define API approval process to allow IT Product teams to expose their products functionality and data via APIs</td>
+    <td>
+    Interoperability<br>
+    EA
+    </td>
+  </tr>
+  <tr>
+    <td>Produce Loose Coupling architecture guidance</td>
+    <td>Produce a guidance document to clarify and guide Architects in building IT solutions making use of loose coupling architecture principles that provide IT Product teams more autonomy (e.g. Micro-Services, 12-factor app)</td>
+    <td>Solution Architecture</td>
+  </tr>
+  <tr>
+    <td>Product Management</td>
+    <td>Get endorsement from the Resource Centre</td>
+    <td>Engage with the Resource Centre to provide an alternative means to support projects, by adopting a team mentality over individuals</td>
+    <td>
+    Resource Centre<br>
+    Senior Advisors
+    </td>
+  </tr>
+  <tr>
+    <td>DevOps</td>
+    <td>Get endorsement from Operations</td>
+    <td>Engage with Enterprise Operations and BOSS to get their endorsement in moving to a DevOps model, in parallel with the conventional ones</td>
+    <td>
+    IT Security<br>
+    Release Services<br>
+    Testing Services<br>
+    Accessibility<br>
+    Environments
+    </td>
+  </tr>
+  </table>
+
+## Measuring the Strategy's success
+
+This Strategy ‘s success will be measured by comparing the following metrics against the conventional IT project methodology.
+
+The metrics are defined as follows:
+
+1. Lead time: the time code changes take to go from check-in to release in production
+2. Deployment frequency: the rate at which software is deployed to production or released to end users
+3. Change fail: the change failure rate measured by how often deployment failures occur in production that require immediate remedy
+4. Time to restore: the time it takes from detecting a user impacting incident to remediating it
+5. Client satisfaction: the general level of contentment, by applications within the APM portfolio, that a client is satisfied with the IT product
```

Comment:
I agree these are awesome metrics, of course. But shouldn't the entire dept be measured against this? Perhaps it belongs in the opening section Guiding Policy ?... or perhaps we need another section for metrics generally for the entire department, and just say that we are accountable for improving those?